playground/repo/pyproject.toml

```
@@ -12,7 +12,7 @@ version = "0.0.1"
 description = "CI tools for Repository Plaground"
 readme = "README.md"
 dependencies = [
-  "securesystemslib[gcpkms, sigstore] @ git+https://github.com//secure-systems-lab/securesystemslib", 
+  "securesystemslib[gcpkms, sigstore] @ git+https://github.com/secure-systems-lab/securesystemslib", 
```

```suggestion
  "securesystemslib[gcpkms, sigstore, pynacl] @ git+https://github.com/secure-systems-lab/securesystemslib", 
```
needed for ed25519, which the tests seem to do